content/pages/errors/403.html

```diff
@@ -0,0 +1,26 @@
+<html>
```

Comment:
This page seem to be exactly the same as 401. Should we create a symlink instead to avoid having to maintain two HTML files with the same content?

Reply:
They are very slightly different for now, but do serve different purposes. 401 is not authorized as in user is not logged in, 403 is access denied as in wrong permisisons. We could make these two more distinct if we want.

Reply:
I updated the 401 error with a little bit more copy. I'm not even sure we differentiate between the two ever in the application either though.